### DISCUSSION TITLE
How to use virtual display for games

### DISCUSSION THREAD

**Discussion by totalitarian** [VirtualDrivers/Virtual-Display-Driver#38] [Q&A]
Sorry for the daft question but moonlight uses my main monitor rather than my virtual display. If I set the virtual display as my main monitor - moonlight correctly uses the virtual display BUT, when I reboot, I get a black screen because windows is now using my virtual display.

Am I missing a trick?

**Comment by Travillion** [comment]
Are you using Sunshine? You can add prep and undo commands to switch the display back and forth. For prep: DisplaySwitch 4 and for undo: DisplaySwitch 1. The first command tells Windows to only use your virtual display, and the second command tells Windows to only use your normal display. Under Display Settings in Windows, your virtual display should be set to only display on 1. This turns it off when not being used. This also means you can't set the display resolution settings (because the screen is inactive). You can set the display settings while streaming the desktop, or perhaps modify the options.txt file to only include the resolution you want the virtual display to offer.
